src/new-client/src/App/Footer/ContactUsButton/ContactUs.tsx

```diff
@@ -14,7 +15,13 @@ export const ContactUs: React.FC = () => (
       <span>+1 929 205 4900</span>
     </div>
 
-    <Input value="sales@weareadaptive.com" readOnly />
+    <Input
```

Comment:
I'm not quite sure why this was an input in the first place. 
Perhaps it could use `Link` instead, similar to the website link below?

Reply:
Fixed